sr/comp/types.py

```diff
@@ -64,6 +64,17 @@ def validate(self, extra_data: ScoreOtherData | None) -> None:
 ScorerType = Type[Union[ValidatingScorer, SimpleScorer]]
 
 
+class ExternalScoreData(TypedDict):
+    """
+    The expected YAML data format in "external" scores files is a single root
+    key 'scores' whose value is a list of mappings compatible with this type.
+    """
+
+    team: str
+    game_points: NotRequired[int]
+    league_points: int
```

Comment:
We could start thinking about something like pydantic for validating these data as we read them in based on the types, though that's out of scope here.

Reply:
Yeah, I've been wondering about having some schema stuff.